packages/docs/src/pages/tutorial/events/basic/index.mdx

```diff
@@ -29,17 +34,17 @@ To better understand what is going on we have included simplified HTML of the ap
 </html>
 ```
 
-
-1. The APIs contain `$` to ensure that your code contains them.
-2. The Optimizer looks for `$` and extracts the function wrapped by `$` into a separate lazy-loadable chunk. (See the documentation to better understand how this is done.)
-3. As part of the SSR the server executes the JSX and notices that there is a click listener. The click listener gets serialized into the `<button>` element as `on:click` attribute. (The unique behavior is that the Qwik knows how to subscribe to events as part of the SSR.)
-4. Qwikloader scripts get inlined into HTML. The Qwikloader script sets up a global listener for all events in the browser. (Qwikloader is about 1kb and executes in about 1ms.)
-5. When a user clicks on the button the Qwikloader intercepts the event and looks for an element with `on:click` attribute.
+1. The APIs contain the `$` to ensure that your code contains them.
```

Comment:
What it is trying to say is that type definition of APIs contain `$` so it is not like you can forget to put it there. 
```suggestion
1. The APIs contain the `$` to ensure that TypeScript can verify that your code usage of the API contains the `$` (it is not optional).
```

Is that better? If it is still confusing, happy to remove it.

Reply:
What about something like...

> Qwik APIs contain the `$` so TypeScript can identify areas of code to isolate for lazy-loading.

?

Just let me know if I'm way off track from your original intent 😀

Reply:
It's more like
```
Qwik APIs contain the `$` so that: 
- Optimizer can transform your code for lazy loading
- Developer knows that magic happens here.

TypeScript is there to ensure that you don't skip any `$`
```

See if you can distill that into the text somehow.